contrib/examples/delete-annotations-simple/resources.yaml

```diff
@@ -0,0 +1,29 @@
+apiVersion: constraints.gatekeeper.sh/v1beta1
+kind: K8sRestrictRoleBindings
```

Comment:
I'd suggest using 2 easier-to-understand Kinds here. Maybe some builtin types like ConfigMap or Pod

Reply:
Acked. Used a ConfigMap and Namespace instead.